src/ConstructiveSolidGeometry/VolumePrimitives/RegularPrism.jl

```diff
@@ -124,7 +124,7 @@ function Dictionary(rp::RegularPrism{N,T, <:Any})::OrderedDict{String, Any} wher
     dict = OrderedDict{String, Any}()
     dict["r"] = rp.r # always a Real 
     dict["h"] = rp.hZ*2
-    if rp.origin != zero(CartesianVector{T}) dict["origin"] = rp.origin end
+    if rp.origin != zero(CartesianPoint{T}) dict["origin"] = Dictionary(rp.origin) end
```

Comment:
Is CartesianZero supposed to be used with primitives or is that just there to convert points into vectors and vice versa?

Reply:
Because if so, this might become something using `iszero`